dart-impl/mpi/src/dart_communication.c

```diff
@@ -701,7 +701,7 @@ dart_ret_t dart_accumulate_blocking_local(
 
   CHECK_UNITID_RANGE(team_unit_id, team_data);
 
-  DART_LOG_DEBUG("dart_accumulate() nelem:%zu dtype:%ld op:%d unit:%d",
+  DART_LOG_DEBUG("dart_accumulate() nelem:%zu dtype:%ld op:%ld unit:%d",
```

Comment:
Here `op` is printed as `%ld` where above it is printed as `%p`.

Reply:
OK done. ;)